acdc-ws/conf/application.conf

```diff
@@ -365,10 +365,24 @@ play.assets {
   urlPrefix = "/assets"
 }
 
-# some http request paths are not interesting to capture metrics such as http duration
-acdc.metrics {
-  enable = true
+com.salesforce.mce.kineticpulse.metrics {
+
+    # User configurable metric endpoint at [endpoint].
+    # If endpoint is null or empty string, metrics will not be collected
+    endpoint = "__metrics"
```

Comment:
this one configures `endpoint`, but in routes it also has `/__metrics`. Which one is being used?

Reply:
The application.conf `endpoint` is more to indicate whether metrics gathering is enabled.  If the config value is empty, it would be as if all the paths are bypassed.  Meaning just the generic hotspot metrics (e.g. `jvm_threads_state`...) are gathered and reported at the routes `__metrics` endpoint.

Reply:
ok, so the value is not used in `kineticpulse`?

Reply:
the application.conf `endpoint` is used in `kineticpulse` to determine if to parse the request to create a metric with appropriate metric labels (path, method).

the routes `endpoint` is not used in `kineticpulse`

Reply:
whether the application.conf `endpoint` has value `__metrics` or `true` has the same effect.  So long as the config value is not empty, `kineticpulse` will parse and generate a metric from incoming request.

Reply:
ok, can we change it to boolean to avoid ambiguity? also it might be a good idea to change `endpoint` to `enabled` to make it clearer.

Reply:
got it.  changed from `endpoint` to `enabled`